### PR TITLE
i18n(zh-cn): Update cli-reference.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update cli-reference.mdx

#### Related issues & labels (optional)

- #6043 
- #6040
- #6109

### CODE REVIEW

**Comment by liruifengv** [comment]
There also has a upstream PR: https://github.com/withastro/docs/pull/6040

**Comment by huyikai** [comment]
> There also has a upstream PR: #6040

感谢补充

**Review by sarah11918 — APPROVED** [review]
Wonderful, thank you!

**Comment by liruifengv** [comment]
There are still some that need to be updated @sarah11918

**Comment by sarah11918** [comment]
ok @liruifengv ! I will remove the `merge queue` label!

**Comment by huyikai** [comment]
Updated @liruifengv  please review

File: src/content/docs/zh-cn/reference/cli-reference.mdx [hunk 1]
```diff
@@ -330,6 +312,22 @@ astro --config config/astro.config.mjs dev
 
 为你的项目配置 [`base`](/zh-cn/reference/configuration-reference/#base)。传递此标志将覆盖 `astro.config.mjs` 文件中的 `base` 值（如果存在）。
 
+### `--port <number>`
+
+指定运行开发服务器和预览服务器的端口。默认为 `4321`。
+
+### `--host [optional host address]`
+
+设置开发服务器和预览服务器应监听的网络 IP 地址（即非 localhost IP）。这在开发过程中对于在本地设备（如移动电话）上测试你的项目非常有用。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/cli-reference.mdx:321]:
```suggestion
设置开发服务器和预览服务器应监听的网络 IP 地址（即非 localhost IP）。这在开发过程中对于在本地设备（如移手机）上测试你的项目非常有用。
```

**Review by liruifengv — APPROVED** [review]
LGTM.

**Pull request merged**